debugging.md

```diff
@@ -255,7 +255,7 @@ rendering tree by calling
 As with `debugDumpApp()`, you can call this more or less any time
 except during a layout or paint phase. As a general rule, calling it
 from a [frame
-callback](https://docs.flutter.io/flutter/scheduler/Scheduler/addPersistentFrameCallback.html)
+callback](https://docs.flutter.io/flutter/scheduler/FrameCallback.html)
```

Comment:
you want https://docs.flutter.io/flutter/scheduler/SchedulerBinding/addPersistentFrameCallback.html here